miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/ProjectRestController.java

```diff
@@ -69,18 +63,36 @@
   @Autowired
   private ProjectService projectService;
   @Autowired
+  private SampleService sampleService;
+  @Autowired
   private LibraryService libraryService;
   @Autowired
+  private PoolService poolService;
+  @Autowired
+  private RunService runService;
+  @Autowired
   private SampleGroupService sampleGroupService;
 
   public void setProjectService(ProjectService projectService) {
     this.projectService = projectService;
   }
 
+  public void setSampleService(SampleService sampleService) {
+    this.sampleService = sampleService;
+  }
+
   public void setLibraryService(LibraryService libraryService) {
     this.libraryService = libraryService;
   }
 
+  public void setPoolService(PoolService poolService) {
+    this.poolService = poolService;
+  }
+
+  public void setRunService(RunService RunService) {
+    this.runService = runService;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment; "runService" already holds the assigned value along all execution paths. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4165)